DocumentService/Startup.cs

```diff
@@ -30,8 +23,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddDbContext<DocumentContext>(options => options.UseNpgsql(Configuration.GetConnectionString("DocumentContext")));
-            services.AddSingleton<IDocumentsInitializer, DocumentsInitializer>();
-          
+            services.AddScoped<IDocumentRepository, DocumentRepository>();
```

Comment:
Do we not want this to be a singleton? As it would only open the connection once

Reply:
Okay, that makes sense